tests/modules/programs/thunderbird/thunderbird.nix

```diff
@@ -65,8 +65,6 @@
     };
   };
 
-  test.stubs.thunderbird = { };
-
   nmt.script = ''
```

Comment:
would it be possible to test it with a plugin that requires native messenging ?

Reply:
I've added a check using a firefox native host. I don't think it matters to the test if it's not TB since they are packaged the same way. (There are no native hosts in nixpkgs or elsewhere that we could use right now. In the future, when NUR or nixpkgs support some native hosts, we may switch to a "proper" one.)